scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -325,21 +325,21 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
 
   {% if variant.category in ("mei", "str", "snv", "cancer") %}
     {% if variant.category == "cancer" %}
-    <a href="{{ url_for('variant.cancer_variant',
+    <a href='{{ url_for('variant.cancer_variant',
```

Comment:
The links will not really work well in the popover anyway, but they will at least be valid html this way.